src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -37,6 +41,22 @@ export class RundownComponent implements OnInit, OnDestroy {
       .then(rundownTimingContextObservable => rundownTimingContextObservable.subscribe(this.onRundownTimingContextChanged.bind(this)))
       .then(rundownTimingContextSubscription => (this.rundownTimingContextSubscription = rundownTimingContextSubscription))
       .catch(error => this.logger.data(error).error('Failed subscribing to rundown timing context changes.'))
+    this.subscribeForEventObserver()
+  }
+
+  private subscribeForEventObserver(): void {
+    this.rindownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownAutoNext(this.setAutoNextStartedToTrueFromEvent.bind(this)))
+    this.rindownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownSetNext(this.setAutoNextStartedToFalseFromEvent.bind(this)))
+    this.rindownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownReset(this.setAutoNextStartedToFalseFromEvent.bind(this)))
+    this.rindownEventSubscriptions.push(this.rundownEventObserver.subscribeToRundownDeactivation(this.setAutoNextStartedToFalseFromEvent.bind(this)))
+  }
+
+  private setAutoNextStartedToTrueFromEvent(): void {
+    this.isAutoNextStarted = true
+  }
+
+  private setAutoNextStartedToFalseFromEvent(): void {
```

Comment:
The `fromEvent` part of the name should not be there because it refers to _where_ the method is called from.
Methods should _not_ know anything outside of it's own scope.
There is nothing stopping from calling these from somewhere there isn't from an Event (there is also nothing Event specific within the methods).

Reply:
Done